app/gui/src/dashboard/providers/SessionProvider/SessionProvider.test.tsx

```diff
@@ -73,13 +75,17 @@ describe('SessionProvider', () => {
   it('Should retrieve the user session', async () => {
     const { getByText } = render(
       <Suspense fallback={<div>Loading...</div>}>
-        <SessionProvider
-          authService={authService}
-          mainPageUrl={mainPageUrl}
-          registerAuthEventListener={registerAuthEventListener}
-        >
-          <div>Hello</div>
-        </SessionProvider>
+        <HTTPClientContext.Provider value={new HttpClient()}>
+          <TextContext.Provider value={proxyRefs(useText())}>
```

Comment:
try avoiding `proxyRefs` outside of provider module, make it part of `useText` or create a dedicated function that returns proxied version.